src/scss/variables.scss

```diff
@@ -69,11 +69,11 @@ $btn-font-weight: 600;
 $lead-font-weight: normal;
 
 $grid-breakpoints: (
-  xs: 0,
-  sm: 1px,
-  md: 2px,
-  lg: 3px,
-  xl: 4px,
+  xs: 300px,
+  sm: 640px,
+  md: 960px,
+  lg: 1200px,
+  xl: 1600px,
```

Comment:
These seems kind of big. Have we considered using the bootstrap default values?

https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L195-L201

Reply:
We can look at other sizing options, but I felt it worked well in this case with the vault items.